packages/block-library/src/paragraph/edit.native.js

```diff
@@ -40,6 +43,15 @@ function ParagraphBlock( {
 	const onAlignmentChange = useCallback( ( nextAlign ) => {
 		setAttributes( { align: nextAlign } );
 	}, [] );
+
+	const parentTextAlignment = allowedParentBlockAlignments.includes(
+		parentBlockAlignment
+	)
+		? parentBlockAlignment
+		: undefined;
+
+	const textAlignment = align || parentTextAlignment;
```

Comment:
I'm only seeing this change for the paragraph block. It looks like the quote block's alignment affects more than just paragraph block children on the web (i.e., heading, list, etc. also get the parent's alignment when they are inside a quote block). Is this an intentional difference?

Reply:
Nice catch, I'll create a ticket to add support for other blocks as well. I think it's fine to ship it supporting the Paragraph block first since it's the default block that the Quote block uses, and supporting the rest for the next release.